src/v/coproc/offset_storage_utils.cc

```diff
@@ -27,76 +28,11 @@
 
 #include <optional>
 
-namespace reflection {
-
-template<>
-struct adl<coproc::ntp_context::offset_tracker> {
-    void to(iobuf& out, coproc::ntp_context::offset_tracker&& offsets) {
-        reflection::serialize(out, static_cast<int32_t>(offsets.size()));
-        for (const auto& e : offsets) {
-            reflection::serialize(
-              out, e.first, e.second.last_read, e.second.last_acked);
-        }
-    }
```

Comment:
is this dead code? it doesn't look related to the subject of the commit

Reply:
`coproc::ntp_context::offset_tracker` is a btree_map, which now can be automatically serialized because of the changes in this PR, so this code is no longer needed